packages/editor/src/client/blocks/richTextBlock/richTextBlock.tsx

```diff
@@ -15,14 +15,24 @@ import {withNormalizeNode} from './editor/normalizing'
 import {LinkFormatButton, RemoveLinkFormatButton} from './toolbar/linkButton'
 import {TableMenu} from './toolbar/tableMenu'
 import {WepublishEditor} from './editor/wepublishEditor'
+import {Alert} from 'rsuite'
 
-export type RichTextBlockProps = BlockProps<RichTextBlockValue>
+export type RichTextBlockProps = BlockProps<RichTextBlockValue> & {
+  maxCharsLimit?: number
+  getCharsLength?: any
```

Comment:
I think the type shoud be: `(text: string) => number` .
But I do not really see why this method is necessary at all?